daemon/daemon.go

```diff
@@ -298,6 +298,7 @@ func (daemon *Daemon) restore() error {
 	wg.Wait()
 	daemon.netController, err = daemon.initNetworkController(daemon.configStore, activeSandboxes)
 	if err != nil {
+		logrus.Error("Error initializing network controller:", err)
```

Comment:
Looks like it's only printed on `stderr`; https://github.com/moby/moby/blob/master/cmd/dockerd/docker.go#L95-L108